### PR TITLE
docs(vs-integration): added dpl wizard article

### PR DESCRIPTION
**Pull request by yanisslav**
Related to: [telerik/blazor##6636](https://github.com/telerik/blazor/issues/6636)

### CODE REVIEW

File: getting-started/vs-integration/dpl-wizard.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+>caption Start the Document Processing Libraries Wizard from the Extensions Menu
+
+![Start the Document Processing Libraries Wizard from the Extensions Menu](images/vs-ext-dpl-wizard-extensions-menu.png)
+
+When you select a package, the main page of the Wizard will appear. In the **SELECT DPL** section, all available packages will be listed, and the one you selected will be highlighted. Clicking the **FINISH** button will install the selected package and add the necessary references to your application.
+
+>caption Document Processing Libraries Wizard Main Page
+
+![Document Processing Libraries Wizard Main Page](images/vs-ext-dpl-main-page.png)
```
Comment (yordan-mitev) [inline getting-started/vs-integration/dpl-wizard.md:51]:
```suggestion
![Document Processing Libraries Wizard main page - using the Telerik UI for Blazor VS extensions](images/vs-ext-dpl-main-page.png)
```

**Review by yordan-mitev — APPROVED** [review]

File: getting-started/vs-integration/dpl-wizard.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+title: Using the Document Processing Libraries Wizard
```
Comment (ntacheva) [inline getting-started/vs-integration/dpl-wizard.md:2]:
```suggestion
title: Using the Document Processing Libraries
```
Remove the "Wizard" word from the titles for consistency. The other article titles do not include it. Applies for page title as well.

File: getting-started/vs-integration/dpl-wizard.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+1. Right-click the project node and select **Telerik UI for Blazor**.
+1. Select the package you want to include.
+
+>caption Start the Document Processing Libraries Wizard from the Context Menu
+
+![Start the Document Processing Libraries Wizard from the context menu of a Telerik UI for Blazor app](images/vs-ext-dpl-wizard-context-menu.png)
+
+### Extensions Menu
+
+Alternatively, instead of the context menu, you can use the VS **Extensions** menu at the top:
```
Comment (ntacheva) [inline getting-started/vs-integration/dpl-wizard.md:36]:
```suggestion
```
No need for that, we are already listing this is the other option.

File: getting-started/vs-integration/dpl-wizard.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+1. In the **Solution Explorer**, select the Blazor app in which you want to add a Telerik DPL package.
```
Comment (ntacheva) [inline getting-started/vs-integration/dpl-wizard.md:38]:
```suggestion
```
I believe this is not part of the process here.

File: getting-started/vs-integration/dpl-wizard.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+## Starting the Wizard
+
+To start the wizard, use either of the following options:
+
+* [The Project Context Menu](#project-context-menu)
+* [The VS Extensions Menu](#extensions-menu)
+
+### Project Context Menu
+
+1. In the **Solution Explorer**, select the Blazor app in which you want to add a Telerik DPL package.
```
Comment (ntacheva) [inline getting-started/vs-integration/dpl-wizard.md:26]:
```suggestion
1. In the **Solution Explorer**, select the Blazor app in which you want to add a Telerik DPL package.
```
```suggestion
1. In the **Solution Explorer**, right-click the Blazor app in which you want to add a Telerik DPL package.
```
To be specific on the action.

**Comment by ntacheva** [comment]
- I suggest moving the article after the "Configuring Existing Telerik UI for Blazor Projects" article, so all Wizards are listed together. Additionally, we may extract them in a dedicated folder "Wizards" (not required now).

- It will be useful to link the article in the cases where clients actually need to install them. For example: https://docs.telerik.com/blazor-ui/knowledge-base/grid-custom-cell-formatting-with-radspreadprocessing

**Comment by ntacheva** [comment]
There are several kb's that show customization using DPL(the linked one was just an example). My idea was to link the Wizard article where we mention installing the package for better visibility.
image

**Review by ntacheva — APPROVED** [review]

**Review by ntacheva — APPROVED** [review]

**Pull request merged**